consensus/hotstuff/committees/leader/cluster.go

```diff
@@ -23,6 +23,7 @@ func SelectionForCluster(cluster protocol.Cluster, epoch protocol.Epoch) (*Leade
 	identities := cluster.Members()
 	seed, err := epoch.Seed(indices.ProtocolCollectorClusterLeaderSelection(cluster.Index())...)
 	if err != nil {
+		return nil, fmt.Errorf("could not get leader selection seed for cluster (index: %v) at epoch: %v: %w", cluster.Index(), counter, err)
```

Comment:
Does it make sense to continue at all here?

Reply:
I don't think so. An error here indicates a critical problem with the `epoch` object we receive as argument